src/components/Input/BoolInput.tsx

```diff
@@ -0,0 +1,33 @@
+import { JSX } from 'react';
+import { Checkbox, Group } from '@mantine/core';
+
+import { Label } from '../Label/Label';
+
+interface Props {
+  name: string;
+  description: string | JSX.Element;
+  value: boolean;
+  setValue: (value: boolean) => void;
+  readOnly?: boolean;
+}
+
+export function BoolInput({
+  value,
+  setValue,
+  name,
+  description,
+  readOnly = false
+}: Props) {
+  return (
+    <Group gap={'xs'} wrap={'nowrap'}>
+      <Checkbox
+        checked={value}
+        onChange={(event) => setValue(event.currentTarget.checked)}
+        onKeyDown={(event) => event.key === 'Enter' && setValue(!value)}
+        disabled={readOnly}
+        aria-label={`Toggle ${name}`}
```

Comment:
I think we might want to put this on each individual component instead, to be able to make sure that it always makes sense. 

Now there will be some case where the aria label reads "Toggle toggle ..."  due to how the bool inputs are named

We could make it a required property of this component? :)

Reply:
Unfortunately, the aria-label can not be passed as a prop unless we rename it to ariaLabel, which would be a new way to pass an aria-label in this repo and I'm not a huge fan of that tbh... Making the aria-label the name for now :) I think that should be good enough